scripts/globals/combat/physical_utilities.lua

```diff
@@ -19,10 +19,58 @@
 
 -- 4  - Add them all, and final operations/considerations.
 -----------------------------------
+require("scripts/globals/items")
+-----------------------------------
 xi = xi or {}
 xi.combat = xi.combat or {}
 xi.combat.physical = xi.combat.physical or {}
 -----------------------------------
+local wsElementalProperties =
+{
+-- [Resonance] = { None, Fire, Ice, Wind, Earth, Thunder, Water, Light, Dark },
+    [       0] = {    0,    0,   0,    0,     0,       0,     0,     0,    0 }, -- Lv 0, NONE
```

Comment:
Seems like there's a lot of excessive whitespace here, I'm fine with it because of the comments, but also don't want to make this a habit of folks

Reply:
Also, should these values be bool instead, or is there a reason to use 0,1?

Reply:
It can be either.

I initially coded it as booleans, but visually, for some reason, it was just easier on the eye, for me, to read the table with 0s and 1s

Reply:
There are unfortunately things that aren't .1 ftp
https://www.bg-wiki.com/ffxi/Gavialis_Helm

Reply:
(maybe I misread, 0,1 as 0.1?)

Reply:
The bonus is added after.
This table is meant to return 0/1 (or false/true, idm) depending on the weaponskill property (fusion, liquefaction, etc...) and the element we are cheking against.
It's a "match" table.